File: gcode/src/main/java/org/qw3rtrun/p3d/g/event/AdvancedOKReceived.java
package org.qw3rtrun.p3d.g.event;

import org.qw3rtrun.p3d.core.msg.AdvancedOkReceivedEvent;

public record AdvancedOKReceived(int planner, int blockQueue, int lineNumber) implements AdvancedOkReceivedEvent {
}

File: gcode/src/main/kotlin/org/qw3rtrun/p3d/g/marlin/event/AdvancedOKReceived.kt
package org.qw3rtrun.p3d.g.marlin.event

import org.qw3rtrun.p3d.core.msg.AdvancedOkReceivedEvent

data class AdvancedOKReceived(
    @get:JvmName("getPlanner") val planner: Int,
    @get:JvmName("getBlockQueue") val blockQueue: Int,
    @get:JvmName("getLineNumber") val lineNumber: Int
) : AdvancedOkReceivedEvent {
    override fun planner(): Int = planner
    override fun blockQueue(): Int = blockQueue
    override fun lineNumber(): Int = lineNumber
}
